backend/Cyberia/urls.py

```diff
@@ -0,0 +1,6 @@
+from django.urls import path
+from Cyberia import views
+
+urlpatterns = [
+    path('Cyberia/', views.Cyberia)
```

Comment:
Nitpick: Can you change the url to lowercase, to follow naming conventions for REST APIs URLS (https://restfulapi.net/resource-naming/)